### ISSUE TITLE
ngc4569

### ISSUE THREAD

**Issue by ashleythomasbarnes** [ashleythomasbarnes/phangs#26]
The QA report checklist for this galaxy
Please complete as you progress through the QA
[All data for galaxies can be obtained here](https://drive.google.com/file/d/1jc1PwXf-U_f_yjCxUl8wrPPE5LzpDzYu/view?usp=sharing)

Checklist
---
* [x] tp - check baseline of single-dish image
* [x] tp - check for artefacts in single-dish image
---
* [x] 7m - check residual map 
* [x] 7m - check baseline of 7m image (i.e. in continuum subtraction okay?)
* [x] 7m - check for negative bowls in 7m image
* [x] 7m - check for artefacts in 7m image (linear features in map, large fluxes etc)
---
* [x] 7m+tp - check baseline of 7m image (i.e. in continuum subtraction okay?)
* [x] 7m+tp - check for negative bowls in 7m image
* [x] 7m+tp - check for artefacts in 7m image (linear features in map, large fluxes etc)
* [x] 7m+tp - compare to the single dish image (fluxes okay?) 
---
* [x] *Dataset science ready?*
---

**Comment by ceibensteiner** [comment]
HCN -- okay
![Bild 07 02 22 um 10 34](https://user-images.githubusercontent.com/56962537/152762193-db946b0a-d6d2-4260-8cf0-e3797c3c18a2.jpg)

**Comment by ceibensteiner** [comment]
HCO+ -- okay
![Bildschirmfoto 2022-02-07 um 10 39 01](https://user-images.githubusercontent.com/56962537/152763342-78ac491f-95d6-4cf7-81d2-075766ee26c3.png)

**Comment by ceibensteiner** [comment]
CS -- weird-looking periodic negative blobs in 7m  and residual
![Bildschirmfoto 2022-02-07 um 10 44 08](https://user-images.githubusercontent.com/56962537/152764233-51ccad13-f218-48a4-8772-792570685f5c.png)

**Comment by ashleythomasbarnes** [comment]
Indeed, setting to pass but should take a look at this CS feature...
